Pilot/Bridge Officer Changes

FearTheBlackout opened the pull request:
<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You find a README and example file in .\html\changelogs\ for further instructions.
-->
In summary:
- The SolGov Pilot has been removed as a role entirely; its responsibilities have been given to the Bridge Officers, who will fly the Torch, Aquila, and Guppy when needed, while the NanoTrasen Pilot is responsible for flying the Charon, Petrov, and Guppy.
- The Bridge Officer now gets the Pilot's headset.
- The Pilot's Qualification Pin is now an item in loadout solely for Bridge Officers.
- The SolGov Pilot's locker has been removed from the Pilot's Lounge.
- The Pathfinder now has access to the Charon helm in the event a NanoTrasen Pilot is not present.

dryerlint commented:
I would suggest maybe renaming the Pilot's headset - how about "helmsman's headset", now that we have that as an alt-title?

N8-Toe commented:
Letting Bridge Officers be above O-1 is a bad move as it makes it so they can be higher ranked than department heads.

ghost commented:
Doesn't matter; their place in the Chain of Command remains the same.

comma commented:
Don't really dig the rank bump for bridge officers, but not really strong feeling about that.
Locking SCG out of Charon is more of a problem IMO, since from what I've seen lately NT doesn't give as much crap about going on away missions, and NT pilots are mostly non-existant. Maybe give Pathfinder the access?

ghost commented:
That could work.

babydoll commented:
Really don't like having a unique alt title on a job with 2 slots, especially when the job you're merging into this one barely has much work to do, anyway.

ghost commented:
Don't see your point; this'll give Bridge Officers more to do between the two of them. They can both be Helmsmen because one might stay behind on the Torch in case of emergency maneuvers while the other is flying the Aquila or Guppy.

Sbotkin commented:
I LIKE making pilot a BO's sub-title, I DISLIKE giving Charon to NTP solely. But, AFAIK, there is no limit for sub-titles, and we don't need more than one pilot (may be wrong about the limit tho). Also, to this point, I want to make Pathfinder a nominal Head of Department (exploring).

Sunbeamstress commented:
Not that I give a shit either way (I rather like this change) but bumping Bridge Officer up to O-2 minimum means we'll lose the xeno BOs that were previously allowed. So this'll probably piss someone off!

ghost commented:
Can they not just continue being Bridge Officers at O-1...?

EDIT: Oh I see what you mean. They can still be O-1 but they can go as high as O-2 for Fleet/Marines and O-3 for EC.

Sunbeamstress commented:
Oh, I misunderstood. I thought you had to be O-2 at minimum now. Nevermind!

Techhead0 commented on maps/torch/items/headsets.dm:
```diff
@@ -45,12 +45,12 @@
-/obj/item/device/radio/headset/pilot
-	name = "pilot's headset"
+/obj/item/device/radio/headset/helmsman
+	name = "helmsman's headset"
```
It's unusual to name the a job's item after the alt-title and not the main role.

FearTheBlackout replied:
Fair enough.

Techhead0 commented on maps/torch/job/jobs.dm:
```diff
@@ -331,13 +332,17 @@
+		/datum/mil_rank/marine/o1,
+		/datum/mil_rank/fleet/o2,
+		/datum/mil_rank/marine/o2,
+		/datum/mil_rank/ec/o3
```
As others have said, I'd feel BO should be kept to a junior officer job, and not equal to a department head. (ie. drop O-3 at the least, maybe drop O2 too and keep it at the status quo)

FearTheBlackout replied:
I'm fine with dropping O-3. Considering Bridge Officers are higher in the Chain of Command than other Staff Officers and a Physician can outrank a Line Officer, I don't see them being O-2 as a problem.

Techhead0 commented on maps/torch/job/outfits.dm:
```diff
@@ -193,10 +193,11 @@ Keeping them simple for now, just spawning with basic EC uniforms, and pretty mu
 /decl/hierarchy/outfit/job/torch/crew/command/bridgeofficer/fleet
 	name = OUTFIT_JOB_NAME("Bridge Officer - Fleet")
-	uniform = /obj/item/clothing/under/solgov/utility/fleet/command
+	uniform = /obj/item/clothing/under/solgov/utility/fleet/command/pilot
```
If the job's primary role and title isn't piloting, I don't think they need a pilot's pin.

FearTheBlackout replied:
Should the pin be left in code, then?

Techhead0 replied:
Maybe in the future we get space pods or whatever and we need the pilot job back, who knows. Or someone gets it in their head to use it for events.

Techhead0 reviewed (commented):
Overall, like the concept, have disagreements on implementation.

Techhead0 commented on maps/torch/job/jobs.dm:
```diff
@@ -321,6 +321,7 @@
 	selection_color = "#2f2f7f"
 	minimal_player_age = 18
 	economic_modifier = 7
+	alt_titles = list("Helmsman")
```
I'm doing this separately because it's optional, not a request...

How do you feel about "Helm Officer"?

FearTheBlackout replied:
I'm fine with it. It would make them clearly distinguished from Enlisted.

EcklesFire commented:
Very much against the rank bump.
Bridge officer is THE junior officer role for getting into officer things. It does not need to be able to outrank/be on par with a line officer above O-1. 

I am not a fan of the helmsman alt title, this just muddies the duties of the role and, suddenly, you end up with two helmsmen joining and both arguing over what they do/don't/want to do with each other and with command.

These two changes, specifically, are muddying the waters and overextending the role. So I am firmly against these two bits.

AtomicWorm commented:
I don't see a reason why this change is necessary.

Also, bumping EC BOs to O-3... why, exactly? They're junior officers. They're meant to be junior officers. The officer roles are already top-heavy as it is.

Techhead0 commented:
They've already been yanked from O-3 as of the latest commit, you two can chill.

SiegDerMaus commented:
I don't like them being able to be O-2 much either.

ThatOneGuy42 commented:
I do not support this. Removing the NT Pilot is a far better route to take, and the rank bump for BOs is not smart decision making.

Spookerton commented:
This is a horrible idea for a large number of reasons relating to separation of roles and general game flow. Do not want. No. Bad.

comma commented:
@Spookerton 
>This is a horrible idea for a large number of reasons relating to separation of roles and general game flow. 

gib some

EcklesFire commented:
@Techhead0 

I still really do not see why they need O-2 bumps.

The entire point of bridge officer is that it is THE junior officer position which you can get some command learning in. It was a solid cap that was expected and lined up with that position. 

I do not feel comfortable with that rank up they're getting for very little reason.

ghost commented:
More Responsibility = More Rank. It isn't fair to let Bridge Officers fly the ship AND have them fairly high in the Chain of Command while keeping them at the lowest rank.

comma commented:
They're rock bottom in Chain of Command, they're artificially high in Chain of Succession. As I said I don't like it myself, and looks like it's generally poorly recieved, so maybe put them back in O1. It would indeed make them have ideas that they have any COMMAND authority, which they certainly do not, they borrow that authority from line officers. They're mouth, not brain.

Also I just got a crazy idea - axe the alt title. Give BO's flight qualification pin as loadout item. If someone wants to underline the fact they're a flyboy, they can get one, otherwise all BOs are assumed to need to take helm as needed.

ghost commented:
OP updated with requested changes.

ThatOneGuy42 commented:
Also still completely against NTP having Charon only that’ll fail spectacularly

ghost commented:
You mean only being able to fly the Charon or nobody but them being able to fly the Charon?
1. The NTP can still fly the Petrov.
2. The Pathfinder will get Charon access as a compromise.

SiegDerMaus commented:
Still don't like this either, even with the O-2 bump removed.

ThatOneGuy42 commented:
FTB I'm against the NTP even *existing*

BlueNexus commented:
Really don't like this, for the same reasons as those contributed by everyone else. Please don't do this - Can we get a contentious label on this?

comma commented:
Alright, that's a lot of skreeching, make a thread on the forums to sort it out.

Crushtoe commented:
For some good friendly screeching, I'm in favor of the PR in its current form, with them remaining O-1. Crushtoe approves.

Commit:
Pilot Changes


Changes "Pilot" to "Helmsman"; gives Pathfinder Charon helm access


Fixes BO ranks


Changes


Revert "Pilot Changes"

This reverts commit 983101535b3bf215c56093b11d41773bd8f11dcd.

Pilot Changes


Changes "Pilot" to "Helmsman"; gives Pathfinder Charon helm access


Fixes BO ranks


Changes

ghost commented:
Ok this is fine now.

comma commented:
Looking at that thread, looks like it all boiled down to concerns over BOs with alt title claiming they're not REALLY BOs, and letting people pick the fancy pin to signify their pilotness seems to satisfy it.
Gonna go ahead and merge it unless someone stops me.

ghost commented:
Last-minute question: What if I made the Pilot's pin available to all Officer roles (or at least Command roles) in the loadout? That way anyone on the Bridge could volunteer to fly.

comma commented:
Eh, let's try this for now.
